03_launch_mgmt_cluster.sh

```diff
@@ -333,21 +280,51 @@ function update_component_image(){
     TMP_IMAGE_TAG="latest"
   fi
 
```

Comment:
What if we used another function `extract_image_name_and_tag`

Reply:
Same as above, I didn't want to change more than necessary since it makes it harder to understand what actually changed.